spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/ReflectUtils.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.utils.intermediate
+
+import com.intel.analytics.bigdl._
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.serializer.ModuleSerializer._
+import scala.collection.mutable
+import scala.reflect.{ClassTag, ManifestFactory}
+import scala.reflect.runtime._
+
+object ReflectUtils {
```

Comment:
RelectionUtils ?

Reply:
Conflict to Scala RelectionUtils, so I use "ReflectUtils"

Reply:
but we can still name it right ? we just need to use different import statement

Reply:
ok, if you persist.

Reply:
ok, chang to RelectionUtils